### ISSUE TITLE
Jaws cant recover after CloudFormation rollback

### ISSUE THREAD

**Issue by Nopik** [serverless/serverless#178]
I had some stack deployed, I manually deleted it from CloudFoundation. Then, I wanted to deploy it again via `jaws dash`. It failed, due to some problems in `resources-cf.json`, which seem to be irrelevant to this issue. Now, CloudFormation stack went into ROLLBACK_COMPLETE status and `jaws dash` cant deploy anymore:

```
JAWS: - { [ValidationError: Stack:arn:aws:cloudformation:us-east-1:151343252086:stack/devkamil-bpg-l/da3f98f0-674a-11e5-aa9a-50e2416294e0 is in ROLLBACK_COMPLETE state and can not be updated.]
  cause:
   { [ValidationError: Stack:arn:aws:cloudformation:us-east-1:151343252086:stack/devkamil-bpg-l/da3f98f0-674a-11e5-aa9a-50e2416294e0 is in ROLLBACK_COMPLETE state and can not be updated.]
     message: 'Stack:arn:aws:cloudformation:us-east-1:151343252086:stack/devkamil-bpg-l/da3f98f0-674a-11e5-aa9a-50e2416294e0 is in ROLLBACK_COMPLETE state and can not be updated.',
     code: 'ValidationError',
     time: Wed Sep 30 2015 10:13:49 GMT+0200 (CEST),
     statusCode: 400,
     retryable: false,
     retryDelay: 30 },
  isOperational: true,
  code: 'ValidationError',
  time: Wed Sep 30 2015 10:13:49 GMT+0200 (CEST),
  statusCode: 400,
  retryable: false,
  retryDelay: 30 }
```

**Comment by boushley** [comment]
Alright, so the core of the problem here is that you can't update a CloudFormation stack that never finished creation. So if a stack reaches `ROLLBACK_COMPLETE` the only way forward is to delete it. We handle `UPDATE_ROLLBACK_COMPLETE` just fine as it stands.

So we should figure out how we want to handle `ROLLBACK_COMPLETE` cases. We would either need to delete the stack, or choose a new name (which doesn't work with the way we currently handle stacks). The simplest solution to this would probably be to prompt the user if we should delete the stack for them or let them handle it.

That said I think it might be worth considering adding the stack name or arn to the stage definition in the `jaws.json` file. This way we could append a random 5 characters or something along those lines. When we run into something like this we could create a new stack instead of being blocked on the existing one.

Both of these options would help resolve this problem. I doubt we'll see this very often, so maybe even just detecting the error and showing a more friendly error message letting the user know they need to go clean up the stack.

**Comment by Nopik** [comment]
Yeah, experienced users wont hit that problem often. Newbies might do it once, though. I'd say that detection of the case + appropriate error message would suffice. Automatic deletion isn't always desirable, too.

**Comment by OneMuppet** [comment]
@Nopik @boushley I had this very problem last night, after a couple of hours I ended up deleting and re-creating my entire project to solve this. In my case I was apply some changes to the dynamoDB which didn't work out, so I used the AWS console to delete the dynamoDB tables manually, after that i received the ROLLBACK_COMPLETE. I didn't find any good way to move forward from there so eventually I deleted everything and started over.

I would love it if jaws: 
- gave me a more friendly message
- gave me an option to delete and/or create a new one

anything is better than just being blocked, eventhough I must say it was a great learning experience having to dig around and actually finding out how things were connected a bit more :)
